README.md

```diff
@@ -1,6 +1,38 @@
-[![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-frontend?branch=develop)
-[![Maintainability](https://api.codeclimate.com/v1/badges/6a767cde4f78e69ece3c/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-frontend/maintainability)
-[![CircleCI](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569.svg?style=svg)](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569)
+[![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-frontend?branch=develop)   [![Maintainability](https://api.codeclimate.com/v1/badges/6a767cde4f78e69ece3c/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-frontend/maintainability)   [![CircleCI](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569.svg?style=svg)](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/develop)
```

Comment:
CircleCi is saying no build what could be the reason?

Reply:
I think CirclrCI has no build because it was configured with `Update circleCI branch` branch instead of `develop branch`